.github/pull_request_template.md

```diff
@@ -13,3 +13,6 @@ Fixes: #issue-number
 ```release-note
 <!-- Enter the release note text here if needed -->
 ```
+
+**How to test (optional)**
+e.g. Run `make tests`
```

Comment:
Would also be nice to mention runtime tests.

Reply:
done!